packages/ui-core/shared/src/lib/tags/tags-color-input/tags-color-input.component.scss

```
@@ -44,7 +44,21 @@ input {
     box-shadow: var(--gauzy-shadow) inset !important;
   }
 }
+
+// The tag name inside a dropdown row. `_overrides.scss` lets an appended
+// `ng-dropdown-panel` grow to `max-content`, so without a cap here ONE
+// pathologically long tag name would decide the width of the whole popup. 20rem
+// is the cap: everything shorter shows in full, only a genuinely long name
+// ellipsises.
+//
+// `min-width: 0` because this span is a flex item, and a flex item's default
```

Stylelint (17.14.1)

[error] Unclosed string (CssSyntaxError)

(CssSyntaxError)